### PR TITLE
[plugin-mobile-app] Add step to send application to the background

### LINKED ISSUE #2841: [Mobile] Add possibility of sending application to the background
**Describe the solution you'd like**
As a user, I want to be able to use a step that sends mobile application to the background. As a workaround emulation of key pressing is used for this purpose now. (e.g. Home);

Example of steps:
```
When I send application to the background for `$duration` seconds
```
**Additional context**
At the moment there is a vividus step for [activating application](https://docs.vividus.dev/vividus/latest/plugins/plugin-mobile-app.html#_activate_mobile_application).
[Appium command](https://appium.io/docs/en/commands/device/app/background-app/) for the functionality.

### PR DESCRIPTION
**Pull request by ikalinin1**
closes #2841

### CODE REVIEW

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java [hunk 1]
```diff
@@ -156,4 +158,15 @@ public void changeAppiumSettings(List<NamedEntry> settings)
 
         execute(executesMethod, entry("setSettings", prepareArguments("settings", prepareArguments(params, values))));
     }
+
+    /**
+     * Run application in background for given period of time.
+     * @param period  Total background run time according to
+     *                <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard
+     */
+    @When("I run app in background for `$period` period")
```
Comment (uarlouski) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java:167]: In all steps that do something with the app we don't use shortcuts in their wording, but I also noticed that there are place with not just application, but mobile application, so I think we should standardise wordings for mobile app steps

File: vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story [hunk 2]
```diff
@@ -240,6 +240,19 @@ When I activate application with bundle identifier `${main-app}`
 When I wait until element located `accessibilityId(menuToggler)` appears
 
 
+Scenario: Verify step: 'When I run app in background for `$period` period'
+When I tap on element located `accessibilityId(menuToggler)`
+When I tap on element located `xpath(<menuScrollViewXpath>)`
+Then number of elements found by `accessibilityId(header)` is equal to `1`
+Then number of elements found by `accessibilityId(footer)` is equal to `0`
```
Comment (uarlouski) [inline vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story:247]: why not to check just header?

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java (hunk 1 above)
Comment (valfirst) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java:167]: `mobile application` -> 
```
When I send mobile application to background for `$period` period
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java (hunk 1 above)
Comment (ikalinin1) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java:167]: @uarlouski  please create an issue to refactor existing steps

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**